docs/App.vue

```diff
@@ -6,6 +6,7 @@
 
 <script>
 import 'semantic-ui-css/semantic.css';
+import 'semantic-ui-css/semantic.js';
```

Comment:
We want to avoid adding other dependencies on top of this

Reply:
What do you propose? As I pointed out to you in PM, shape component require jQuery and SUI.js to work.

Reply:
Yes, we need to reimplement it. I'll try to help you